indico/htdocs/js/indico/modules/category_management.js

```diff
@@ -99,5 +110,53 @@
                 error: handleAjaxError
             });
         }
+
+        function updateCategoryDeleteButton() {
+            if ($table.find(categoryRowSelector).length) {
+                $('.banner .js-delete-category').addClass('disabled');
+            } else {
+                $('.banner .js-delete-category').removeClass('disabled')
+                    .attr('title', $T.gettext("Delete category"));
+            }
+        }
+
+        function bulkDeleteButtonTooltipContent() {
+            var $checked = getSelectedRows();
+            if ($checked.length) {
+                if ($bulkDeleteButton.hasClass('disabled')) {
+                    return $T.gettext("At least one selected category cannot be deleted because it is not empty.");
+                } else {
+                    return $T.gettext("Delete {0} selected categories").format($checked.length);
+                }
+            } else {
+                return $T.gettext("Select the categories to delete first.");
+            }
+        }
+
+        function bulkDeleteCategories() {
+            var selectedCategories = getSelectedCategories();
+            ajaxDialog({
+                url: $table.data('bulk-delete-url'),
+                title: $T.gettext("Delete categories"),
+                data: {
+                    category_id: getSelectedCategories()
+                },
+                onClose: function(data) {
+                    if (data && data.success) {
+                        getSelectedRows().remove();
+                    }
+                }
+            });
+        }
+
+        function getSelectedCategories() {
+            return $table.find(categoryRowSelector).find('input[type=checkbox]:checked').map(function() {
+                return $(this).attr('value');
+            }).toArray();
+        }
+
+        function getSelectedRows() {
+            return $table.find(categoryRowSelector).has('input[type=checkbox]:checked');
```

Comment:
From [jQuery's documentation](https://api.jquery.com/checkbox-selector/):
```
Because :checkbox is a jQuery extension and not part of the CSS specification,
queries using :checkbox cannot take advantage of the performance boost
provided by the native DOM querySelectorAll() method. For better performance
in modern browsers, use [type="checkbox"] instead.
```

Also with `[type=checkbox]` you don't have to: look at the CSS spec, find nothing, wonder why it appear to work in all browsers and then find out after searching the web that it is a jQuery extension that has for only advantage to allow one to spare 5 characters.

I'd prefer to keep the standard `[type=checkbox]` since the jQuery addition brings nothing interesting.